client/.eslintrc

```diff
@@ -2,10 +2,10 @@
   "parser": "babel-eslint",
   "extends": "airbnb",
   "settings": {
-    import/resolver: "webpack"
+    "import/resolver": "webpack"
```

Comment:
In both pre and post update, eslint cannot resolve `lib` without the config. I get errors like: 
```
  1:1   error  'lib' should be listed in the project's dependencies. Run 'npm i -S lib' to add it  import/no-extraneous-dependencies
  1:33  error  Unable to resolve path to module 'lib/helpers/table_row_form_helpers'               import/no-unresolved
  1:33  error  Missing file extension for "lib/helpers/table_row_form_helpers"                     import/extensions
```

You don't get them when you remove the setting and run `npm run lint`?